tailwind.config.js

```diff
@@ -44,6 +44,8 @@ module.exports = {
           '30a': '#EAEBED',
           '40': '#DDDDDD',
           '50a': '#CFD1D5',
+          'blue': '#EBF2FC',
```

Comment:
would you be able to move this to the bright-blue section with maybe something like pale: #hex? 
so it would be added as bright-blue-pale or whatever.